prowler/lib/scan/scan.py

```diff
@@ -296,12 +296,16 @@ def scan(
                         self.get_completed_checks(),
                     )
 
-                    findings = [
-                        Finding.generate_output(
-                            self._provider, finding, output_options=None
-                        )
-                        for finding in check_findings
-                    ]
+                    findings = []
+                    for finding in check_findings:
+                        try:
+                            findings.append(
+                                Finding.generate_output(
+                                    self._provider, finding, output_options=None
+                                )
+                            )
+                        except Exception:
+                            continue
```

Comment:
Same as my previous comment

Reply:
I added that log line within the `generate_output` `try/except` clause, if it is not enough I can add another one here.

Reply:
I didn't notice, its ok for me